apps/arweave/src/ar_chunk_storage.erl

```diff
@@ -333,16 +348,17 @@ handle_cast(do_prepare_replica_2_9, State) ->
 			false ->
 				false
 		end,
+	%% XXX: SubChunkStart2 will always be equal to SubChunkStart, right?
```

Comment:
I kept it from the sub-chunk based version where the increments were of the sub-chunk size. Also, I kept the two-element cursor format such that it will be slightly easier to re-introduce sub-chunk based iterations if we want it (e.g., might be helpful to support low-memory setups.)

Reply:
Got it! I'll update the comment.